packages/adapter/addon/rest.js

```diff
@@ -1067,9 +1062,19 @@ const RESTAdapter = Adapter.extend(BuildURLMixin, {
   },
 
   _ajax(options) {
-    if (get(this, 'useFetch')) {
+    if (this.useFetch) {
       this._fetchRequest(options);
-    } else if (get(this, 'fastboot.isFastBoot')) {
+    } else if (get(this, 'fastboot.isFastBoot') && DEPRECATE_NAJAX) {
+      if (has('fetch')) {
+        deprecate(
+          'You have ember-fetch and jquery installed. To use ember-fetch, set `useFetch: true` in your adapter.  In 4.0, ember-data will fallback to ember-fetch instead of najax when both of these are installed.',
+          false,
+          {
+            id: 'ember-data:najax-fallback',
+            until: '4.0',
+          }
+        );
```

Comment:
I'm assuming we still want the deprecation since, from what I read, the debug-macros are opt-in (no default by reading the current installed e-d version and stripping away deprecations automatically)